Tella/Scenes/Settings/Views/Servers/EditSettingsServerView.swift

```diff
@@ -19,28 +19,29 @@ struct EditSettingsServerView: View {
     var body: some View {
         
         ContainerView {
-            
-            VStack {
-                
-                editServerHeaderView
-                
-                SettingsCardView(cardViewArray: [serverNameView.eraseToAnyView(),
-                                                 serverURLView.eraseToAnyView(),
-                                                 serverUsernameView.eraseToAnyView()
-                                                ])
-                
-                SettingsCardView(cardViewArray: [
-                    AutoUploadView(autoUpload: $serverViewModel.autoUpload,
-                                   isDisabled: serverViewModel.isAutoUploadServerExist).eraseToAnyView(),
-                    $serverViewModel.autoUpload.wrappedValue ? AutoDeleteView(autoDelete: $serverViewModel.autoDelete).eraseToAnyView() : nil,
-                    BackgroundUploadView(backgroundUpload: $serverViewModel.backgroundUpload).eraseToAnyView()
-                    // ShareInfoView(shareInfo: $serverViewModel.activatedMetadata).eraseToAnyView()
-                ])
-                
-                
-                Spacer()
-                
-                bottomView
+            ScrollView {
+                VStack {
+                    
+                    editServerHeaderView
```

Comment:
In my opinion, it's better to exclude the `editServerHeaderView` from the  `ScrollView`

Reply:
you are right! I also excluded the bottom view so the only thing that would be scrollable is the content!